python/plugins/grassprovider/grass_provider.py

```diff
@@ -153,27 +153,18 @@ def svgIconPath(self):
     def versionInfo(self):
         return GrassUtils.installedVersion() or None
 
-    def defaultVectorFileExtension(self, hasGeometry=True):
-        # By default,'gpkg', but if OGR has not been compiled with sqlite3, then
-        # we take "SHP"
-        if 'GPKG' in [o.driverName for o in
-                      QgsVectorFileWriter.ogrDriverList()]:
-            return 'gpkg'
-        else:
-            return 'shp' if hasGeometry else 'dbf'
-
     def supportsNonFileBasedOutput(self):
         """
         GRASS Provider doesn't support non file based outputs
         """
         return False
 
     def supportedOutputVectorLayerExtensions(self):
-        # We use the same extensions than QGIS because:
+        # We use the same extensions as QGIS because:
         # - QGIS is using OGR like GRASS
         # - There are very chances than OGR version used in GRASS is
```

Comment:
Maybe take the opportunity to fix this comment:

```suggestion
        # - There are very few chances than OGR version used in GRASS is
```

I think, a word is missing here.

Reply:
Good catch, missed that one!